### ISSUE TITLE
Problems panel lacks focusable elements

### ISSUE THREAD

**Issue by barmac** [camunda/camunda-modeler#4368]
### What should we do?

The problems noted in the problems panel cannot be focussed with keyboard.

<img width="1136" alt="image" src="https://github.com/camunda/camunda-modeler/assets/28307541/f7253c79-d023-4c24-89c0-1488afc315f2">

```json
{
  "id": "scrollable-region-focusable",
  "impact": "serious",
  "tags": [
    "cat.keyboard",
    "wcag2a",
    "wcag211",
    "wcag213",
    "TTv5",
    "TT4.a",
    "EN-301-549",
    "EN-9.2.1.1",
    "EN-9.2.1.3"
  ],
  "description": "Ensure elements that have scrollable content are accessible by keyboard",
  "help": "Scrollable region must have keyboard access",
  "helpUrl": "https://dequeuniversity.com/rules/axe/4.9/scrollable-region-focusable?application=axeAPI",
  "nodes": [
    {
      "any": [
        {
          "id": "focusable-content",
          "data": null,
          "relatedNodes": [],
          "impact": "serious",
          "message": "Element should have focusable content"
        },
        {
          "id": "focusable-element",
          "data": null,
          "relatedNodes": [],
          "impact": "serious",
          "message": "Element should be focusable"
        }
      ],
      "all": [],
      "none": [],
      "impact": "serious",
      "html": "<div class=\"panel__inner\">",
      "target": [
        ".panel__inner"
      ],
      "failureSummary": "Fix any of the following:\n  Element should have focusable content\n  Element should be focusable"
    }
  ]
}
```

### Why should we do it?

Achieve WCAG 2.1 compliance (a11y)
